packages/astro/src/default/utils/content.ts

```diff
@@ -274,9 +287,11 @@ export async function getTutorial(): Promise<Tutorial> {
         href: joinPaths(baseURL, `/${partSlug}/${chapterSlug}/${nextLesson.slug}`),
       };
     }
-  }
 
-  // console.log(inspect(_tutorial, undefined, Infinity, true));
+    if (lesson.data.editPageLink && typeof lesson.data.editPageLink === 'string') {
+      lesson.editPageLink = lesson.data.editPageLink.replace(':path', lesson.filepath);
```

Comment:
You might need to export the `interpolateString` utility from `@tutorialkit/components-react` and then use it here.

Actuaclly, I'm starting to think that we should move the `interpolateString` utility in the `@tutorialkit/types` package. Wdyt?